app/screens/home/channel_list/categories_list/categories/body/index.ts

```diff
@@ -93,11 +131,16 @@ const getSortedChannels = (database: Database, category: CategoryModel, locale:
                 concatAll(),
             );
         }
-        default:
-            return category.myChannels.observeWithColumns(['last_post_at']).pipe(
-                map(getChannelsFromRelation),
-                concatAll(),
+        default: {
+            const currentUserId = observeCurrentUserId(database);
+            const myChannels = category.myChannels.observeWithColumns(['last_post_at']);
+            const channels = myChannels.pipe(
```

Comment:
basically this is what `getChannelsFromRelation` is doing, not sure why you are changing this

Reply:
I need channel and myChannel information at the same time to reach createdAt field so I did like that